src/content/docs/en/upgrade-astro.mdx

```diff
@@ -180,3 +180,24 @@ A major release may also include some non-breaking changes and improvements that
   Once these features are out of their experimental period, they will follow the normal semantic versioning contract.
 
 - **Improvements to the documentation** (e.g. reference and error messages). They are built from source for the `docs` repository. This allows Astro to quickly update docs fixes and improvements in the cases where documentation source content is stored in the main `astro` repository.
+
+### Node.js support and upgrade policies
+
+#### Support
+
+- Astro supports the **latest _extended_ LTS** versions of Node.js.
```

Comment:
What does "extended LTS" stand for here? On Node's site, they have "Active LTS" and "Maintenance LTS". I suppose this refers to "Maintenance LTS", the longest of the two.
Or is this referring to the extended support that some, like Meteor.js, offer one year beyond the end of the official support from the Node team?

Also, should we link to the Node page so the user can see which version this refers to? I'm guessing that not every user reading this will know from memory which Node version is currently in each support stage.

Reply:
I will update the terminology, using _Maintainance LTS_

Reply:
I already suggested linking to the node versions diagram page and still think it's a good idea

Reply:
Can someone drop the link here to the thing you want to link to?

Reply:
https://nodejs.org/en/about/previous-releases#release-schedule